contracts/starknet/space/space.cairo

```diff
@@ -24,13 +29,16 @@ end
 func proposal_threshold() -> (threshold : Uint256):
 end
 
-# TODO: Should be Address not felt
 @storage_var
-func voting_strategy() -> (strategy_address : felt):
+func voting_strategies(voting_strategy_contract : felt) -> (is_valid : felt):
 end
 
 @storage_var
-func authenticator() -> (authenticator_address : felt):
+func authenticators(authenticator_address : felt) -> (is_valid : felt):
+end
+
+@storage_var
+func executor() -> (executor_address : felt):
```

Comment:
Executor is set per space now, but I think we could imagine having an `executor` per proposal (we'd simply need to change this to take as input a `proposal_id`)